aviary/roost/model.py

```diff
@@ -233,4 +233,7 @@ def forward(
         return torch.mean(torch.stack(head_fea), dim=0)
 
     def __repr__(self) -> str:
-        return self.__class__.__name__
+        return (
```

Comment:
@CompRhys Let me know if there are more/other params you think are important that should be included. Esp. here for Roost...